workshop/src/main/java/nl/jcore/java8workshop/Exercise002ParameterNamesDto.java

```diff
@@ -12,7 +10,7 @@
  * Maven goal "clean" and run the plugins compiler:compile
  * and compiler:testCompile.
  */
-@Immutable
+/* @Immutable */
```

Comment:
Why comment it out? What value does this have as a comment? Maybe it should be removed altogether by the way. Doesn't add that much value to the example.